src/main/drivers/camera_control.c

```diff
@@ -95,14 +114,45 @@ static float calculatePWMDutyCycle(const cameraControlKey_e key)
     return voltage / ADC_VOLTAGE;
 }
 
+static void dlay(const uint32_t ticks) {
```

Comment:
That's not final, pushed it for one person to be able to test tonight. I'm not sure this function will remain at all after I'll find how to use a basic timer for this task.